config/intellij-idea-inspections.xml

```diff
@@ -629,8 +629,8 @@
                    enabled_by_default="false"/>
   <!-- this rule is inspection is only for old java, we are ok to use annotations -->
   <inspection_tool class="Annotation" enabled="false" level="ERROR" enabled_by_default="false"/>
-  <inspection_tool class="AnnotationClass" enabled="true" level="ERROR"
-                   enabled_by_default="true"/>
+  <inspection_tool class="AnnotationClass" enabled="false" level="ERROR"
+                   enabled_by_default="false"/>
```

Comment:
Please put comment on reason of disablement

Reply:
Done

Reply:
"this rule is inspection is only for old java" -> "this inspection is only for old java"?